src/Common/DocumentUtility.ts

```diff
@@ -0,0 +1,10 @@
+import { DefaultAccountExperienceType } from "../DefaultAccountExperienceType";
+import { userContext } from "../UserContext";
+
+export const getEntityName = (): string => {
```

Comment:
It looks like the strings returned are slightly different. `getLabel` returns a string with the first letter capitalized and the word is also pluralized.

We can definitely still unify them but it would take some effort. I can do this in a separate PR.